src/theme/Root.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { useLocation } from '@docusaurus/router';
+import Banner from '../components/banner/Banner';
+
+// The root view wrapper for the entire app
+export default function Root({ children }: { children: React.ReactNode }) {
+  const isInDevelopersSection = useLocation().pathname.startsWith('/developers');
```

Comment:
I dont like this since it feels very hacky, but it works and I couldnt find a better way to do this.

Happy for any feedback if anyone knows a better way to do this

Reply:
First time I'm seeing this repo, but couldn't it go in the index for the developers section here?
https://github.com/jellyfin/jellyfin.org/blob/developers/content/developers/pages/index.tsx

I think that gets rendered when the route is under `/developers`, so we wouldn't need to check the base path in the location then.

Reply:
Oh wait, I see. There's a `pages` and `content/docs` under developers. I wonder if the `docsRootComponent` config option might be a way to do this type of thing?
https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#docsRootComponent

Reply:
Sure, but then this wont show under this page, and ill still be left with the same issue trying to determine which is the normal and which is the developers
![image](https://github.com/jellyfin/jellyfin.org/assets/26843204/ab6dc51d-ad7b-4bc7-867a-3b3ca0d04ad7)

Reply:
Yeah this is the main concern I had. Ideally we'd be able to somehow know we're in the developer part of the site without looking at the path, but I guess we can use this for now. The banner is meant to be temporary anyway.